### PR TITLE
Fix using debug toolbar with latest Django

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/settings.py.dev [hunk 1]
```diff
@@ -63,6 +63,7 @@ INSTALLED_APPS = INSTALLED_APPS + ('storages',)
 # In development, add in extra logging for exceptions and the debug toolbar
 # -----------------------------------------------------------------------------------
 MIDDLEWARE_CLASSES = ('temba.middleware.ExceptionMiddleware',) + MIDDLEWARE_CLASSES
+# MIDDLEWARE_CLASSES = ('debug_toolbar.middleware.DebugToolbarMiddleware',) + MIDDLEWARE_CLASSES
```
Comment (rowanseymour) [inline temba/settings.py.dev]: Not activated by default

Comment (norkans7) [inline temba/settings.py.dev]: the MIDDLEWARE_CLASSES will be duplicated when uncommented

Comment (rowanseymour) [inline temba/settings.py.dev]: Both of these assignments prepend to `MIDDLEWARE_CLASSES` so no duplication

Comment (norkans7) [inline temba/settings.py.dev]: Ohh right

**Pull request merged**